src/Router.cpp

```diff
@@ -110,50 +130,66 @@ void Router::splitTarget(const std::string &target)
 	}
 }
 
-bool Router::pathIsValid(HTTPRequest &request, std::string webRoot)
+bool Router::pathIsValid(HTTPRequest &request, std::string &webRoot)
 {
 	std::string host = request.getHost();
+	std::cout << "pathIsValid Host: " << host << std::endl;
+	std::cout << "pathIsValid WebrRoot: " << webRoot << std::endl;
 	size_t pos = host.find(":");
```

Comment:
Can we start to use the Debug::log and add a ROUTER enum to print stuff in the Router?

Reply:
It is only for development atm ... what do you mean for the parser, 404?